src/index.html

```diff
@@ -6,7 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>DIM</title>
     <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
```

Comment:
not sure if this is a good idea or not...

Reply:
I don't think so. I know iOS Safari just ended up ignoring this because people abused it.

Reply:
ok, removed it